local-volume/README.md

```diff
@@ -80,15 +80,13 @@ unless you want to customize the provisioner spec or storage classes.
 
 ``` console
 $ NODE_LOCAL_SSDS=<n> cluster/kube-up.sh
-$ kubectl create -f provisioner/deployment/kubernetes/admin_account.yaml
```

Comment:
do we still need to keep `admin_account.yaml`? The file is just relocated.

Reply:
I moved it into a generic "examples" directory.  It's just there for reference if you don't use the helm chart to generate all the specs together.  I guess I could remove everything all together and put a sample spec using helm chart defaults to avoid confusion